CMakeLists.txt

```diff
@@ -179,17 +187,18 @@ if(ROS_BUILD_TYPE STREQUAL "catkin")
       find_package(GTest REQUIRED)
     endif()
     find_package(rostest REQUIRED)
+    find_package(Boost REQUIRED COMPONENTS system)
```

Comment:
Should we do this conditionally based on the flags chosen?

Reply:
We could, but ROS1 depends on boost anyways, so it does not hurt to have it here